internal/instance/resource_instance_test.go

```diff
@@ -1893,3 +1908,23 @@ resource "lxd_instance" "instance1" {
 }
 	`, projectName, instanceName, acctest.TestImage)
 }
+
+func testAccInstance_timeout(instanceName string) string {
+	return fmt.Sprintf(`
+resource "lxd_instance" "instance1" {
+  name  = "%s"
+  image = "%s"
+
+  execs = {
+    "0" = {
+      command       = ["sleep", "infinity"]
```

Comment:
I wouldn't go with `infinity` here as any failure will keep a CI runner for no reason. `sleep 30` feels safer.